test/rust-test-util/test-result-reporting-client/Cargo.toml

```diff
@@ -10,6 +10,7 @@ chrono = "0.4"
 enumset = "1.0"
 hyper = { version = "0.14", features = ["client", "http1", "tcp"] }
 thiserror = "1.0"
+tokio = { version = "1", features = ["rt-multi-thread", "macros", "signal"] }
```

Comment:
Surprised I needed to pull in `tokio` just for an async delay. Lmk if alternatives.